### DISCUSSION TITLE
turbulence model

### DISCUSSION THREAD

**Discussion by cczz8** [mdolab/dafoam#688] [Tutorials]
How to modify the turbulence model in DAFoam and compile a new turbulence model？

**Comment by friedenhe** [comment]
Can you be more specific, what do you want to modify? Add a new term?

**Reply by cczz8** [reply]
Dear Professor He Ping,

Thank you for your response. We would like to add the kOmegaSSTSAS turbulence model to adjoint\DAModel and compile it.

**Comment by friedenhe** [comment]
Then, you need to add a child-class to the DATurbulenceModel class. Please refer to how kOmegaSST is added, you can follow a similar approach.
